portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java

```diff
@@ -1,18 +1,50 @@
 package com.google.sps.servlets;
 
 import java.io.IOException;
+import java.util.Arrays;
+import java.util.ArrayList;
+import com.google.gson.Gson; 
```

Comment:
move com.google to the top.

The imports should be in alphabetical order. This makes it easy to scan and quickly find things. Also keep all imports together without any blank lines in between